docker/env/version

```diff
@@ -1 +1 @@
-1.38-argus-backport-perf-v14
+1.39-argus-update-0.11.7
```

Comment:
Current `Hydra` image for the `master` branch of the SCT is `1.54-update-k8s-components-1`.
This one is made for the `perf-v14` specifically.
So, please update it the way we could easily distinguish the master and perf branch hydra images.

Reply:
Done